exatomic/adf/tests/test_output.py

```diff
@@ -24,35 +24,35 @@ def setUp(self):
     def test_parse_atom(self):
         self.lu.parse_atom()
         self.assertEqual(self.lu.atom.shape[0], 1)
-        self.assertTrue(np.all(pd.notnull(pd.DataFrame(self.lu.atom))))
+        self.assertTrue(np.all(pd.notnull(pd.DataFrame(self.lu.atom))).all())
```

Comment:
Do we need both `np.all()` and `.all()` ?

Reply:
Will try enforcing numpy version and removing the workaround.